src/Microsoft.Framework.PackageManager/List/DependencyListCommand.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Versioning;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.PackageManager.List
+{
+    internal class DependencyListCommand
+    {
+        private readonly DependencyListOptions _options;
+
+        public static void Initialize(CommandLineApplication c, Reports reports)
+        {
+            c.Description = "Print the dependencies of a given project.";
+            var runtimeFolder = c.Option("--runtime",
+                "A path to the folder contains framework assemblies. The framework projects usually depend on assemblies in this folder. The command will try to find if there is an assembly in this folder bear the same name to resolve dependencies.",
```

Comment:
I'll suggest some new text for these commands. If I'm way off, then please let me know :smile: 

`A path to the folder containing the framework assemblies to check for <something???>. The framework projects <what framework projects???> usually <usually? when?> depend on assemblies in this folder. The command will try <try? when does it fail?> to find if there is an assembly in this folder with the same name to resolve dependencies.`

OK honestly I still don't understand what this options does. When do I specify it? How do I know what to put in here?